shared/validation/languages/ur.js

```diff
@@ -0,0 +1,50 @@
+import tokenizeWords from 'talisman/tokenizers/words';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 25;
+
+// Numbers that are not allowed in a sentence depending on the language. For
+// English this is 0-9 once or multiple times after each other.
+const NUMBERS_REGEX = /[۱۲۳۴۵۶۷۸۹۰0-9]+/;
+
+// Some languages want to check the structure, this is what this REGEX is for. For English
+// (and by extend as default) we're not currently using it.
+/* eslint-disable-next-line no-unused-vars */
+const STRUCTURE_REGEX = undefined;
+
+// The following symbols are disallowed, please update here as well and not just the regex
+// to make it easier to read:
+// < > + * \ # @ ^ [ ] ( ) ؄ ؃ ؂ ؏ ؆ ؇ ؈ ؁ ٪ ؉ ؊ ؔ ﷽ ؍ ؑ ؐ ﷺ ؒ ﷻ ۘ ۙ ۜ ۚ ۗ ۖ  ٝ ۢ !/ 
+/* eslint-disable-next-line no-useless-escape */
+const SYMBOL_REGEX = /[<>\+\*\\#@\^\[\]\(\)\؄\؃\؂\؏\؆\؇\؈\؁\٪\؉\؊\ؔ\﷽\؍\ؑ\ؐ\ﷺ\ؒ\ﷻ\ۘ\ۙ\ۜ\ۚ\ۗ\ۖ\٘\ٝ\ۢ\!\/]/;
```

Comment:
I want to remove some symbol from here. How can I do this? I didn't find edit option.